lib/modules/ai_helper/entry_point.rb

```diff
@@ -12,8 +12,7 @@ def inject_into(plugin)
         plugin.register_seedfu_fixtures(
           Rails.root.join("plugins", "discourse-ai", "db", "fixtures", "ai_helper"),
         )
-
-        additional_icons = %w[discourse-sparkles spell-check language]
+        
```

Comment:
```suggestion
        
additional_icons = %w[spell-check language]
```

This still needs to keep the "spell-check" and "language" icons. This `%w` syntax converts the contents inside the square brackets into an array, using space as the delimiter.

Reply:
Oh I see, I was thinking ``discourse-sparkles(spell-check, language)``, as I don't speak Ruby.